deploy/monitoring/hosts

```diff
@@ -0,0 +1,2 @@
+monitoring ansible_ssh_host=178.62.116.103 ansible_user=root
+app ansible_ssh_host=138.197.190.164 ansible_user=root
```

Comment:
Let's switch monitoring to the Ship application server, right now it seems to be pointing to the Paralect Platform staging.